src/lib/repository/DeviceRepository.js

```diff
@@ -138,9 +143,16 @@ class DeviceRepository {
 
   callFunction= async (
     deviceID: string,
+    userID: string,
     functionName: string,
     functionArguments: Object,
   ): Promise<*> => {
+    const doesCoreBelongsToUser =
```

Comment:
I would rather have another function called `doesUserHaveAccess` on the deviceAttributeRepository